lib/gooddata/models/synchronization_result.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2015 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require_relative './synchronization_result_details'
+
+require_relative '../mixins/data_property_reader'
+require_relative '../mixins/links'
+
+require_relative '../rest/resource'
+
+module GoodData
+  class SynchronizationResult < Rest::Resource
```

Comment:
Could we name this a little bit more specific. Something like "Client synchronization result" maybe?